kernel/src/main/java/org/kframework/backend/kore/ModuleToKORE.java

```diff
@@ -114,6 +120,10 @@ public String convert(boolean heatCoolEq) {
         collectionSorts.add("MAP.Map");
         collectionSorts.add("LIST.List");
         collectionSorts.add("ARRAY.Array");
+        if (attributes.containsKey("token")) {
+            assert(!attributes.containsKey(HAS_DOMAIN_VALUES));
```

Comment:
I'm reasonably sure this assertion could fail if the user added the attribute to their definition. In general it should be considered a bug for any k definition to generate an assertion failure when compiled

Reply:
Fixed.